packages/mui-material/src/StepIcon/StepIcon.test.js

```diff
@@ -2,14 +2,15 @@ import * as React from 'react';
 import { expect } from 'chai';
 import { createRenderer } from '@mui/internal-test-utils';
 import StepIcon, { stepIconClasses as classes } from '@mui/material/StepIcon';
+import SvgIcon from '@mui/material/SvgIcon';
 import describeConformance from '../../test/describeConformance';
 
 describe('<StepIcon />', () => {
   const { render } = createRenderer();
 
   describeConformance(<StepIcon icon={1} />, () => ({
     classes,
-    inheritComponent: 'svg',
+    inheritComponent: SvgIcon,
```

Comment:
this change is to display `Props of the SvgIcon component are also available.` in props table. 

https://deploy-preview-44337--material-ui.netlify.app/material-ui/api/step-icon/#props